etl/src/main/scala/etl/ETL.scala

```diff
@@ -69,16 +88,17 @@ object ETL extends App {
       println("Articles with no recipes:")
       endResult.articlesWithNoRecipes.foreach(id => println(s"- https://www.theguardian.com/$id"))
 
-      //add images for articles with recipes and no images
-      def articlesWithoutImages: Set[String] = db.getArticlesWithRecipes.toSet diff db.getArticlesWithSavedImages.toSet
-      articlesWithoutImages.foreach { articleId =>
-        capiClient.getResponse(ItemQuery(articleId)).foreach { itemResponse =>
-          db.insertImages(ImageExtraction.getImages(itemResponse.content).toList)
+    } finally {
+      val fArticles: Future[List[SearchResponse]] = Images.getArticlesMissingImages()
```

Comment:
It feels odd to do this work in a `finally` block. Have you already spoken to @mchv about this given he's in the office?